controller/replica_controller.go

```diff
@@ -273,11 +273,11 @@ func (rc *ReplicaController) syncReplica(key string) (err error) {
 
 	log := getLoggerForReplica(rc.logger, replica)
 
+	if !rc.isResponsibleFor(replica) {
+		// Not ours
```

Comment:
consider removing the comment, doesn't really add anything that is not expressed by.
We are not responsible for this :)

Reply:
Thank you. Fixed